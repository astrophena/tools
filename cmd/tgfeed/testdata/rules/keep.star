# Â© 2024 Ilya Mateyko. All rights reserved.
# Use of this source code is governed by the ISC
# license that can be found in the LICENSE.md file.

feeds = [
    feed(
        url = "https://example.com/feed.xml",
        keep_rule = lambda item: "keep" in item.title.lower(),
    )
]
