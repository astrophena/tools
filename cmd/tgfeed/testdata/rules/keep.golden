[
  {
    "chat_id": "test",
    "entities": [
      {
        "length": 14,
        "offset": 3,
        "type": "text_link",
        "url": "https://example.com/keep"
      }
    ],
    "link_preview_options": {
      "is_disabled": false
    },
    "reply_markup": {
      "inline_keyboard": [
        []
      ]
    },
    "text": "ðŸ”— Keep this item\n#examplecom\n\n"
  }
]